src/libraries/listener.library.ts

```diff
@@ -1,11 +1,29 @@
 import { BlackListedKeyboardShortcutChecker } from './black-listed-key-bindings.library';
 
-export interface IKeyboardShortcutListenerOptions {
+export interface IKeyboardShortcutListenerOptions { // defaults with ?
```

Comment:
there is a comment you left here "// defaults with ?" that I think you put in as a note to yourself.  Go back through and see if you find others like this too